### ISSUE TITLE
Fluid property range with different unit than SI in log file

### ISSUE THREAD

**Issue by tboussaid** [oemof/tespy#417]
Hi,
First of all, thanks for the amazing work put in this library. I am using it for my PhD project.
For convergence reasons, it's often better to set fluid properties ranges. However, I was going over the logg file, and I just noticed a small 'mistake' when the unit of the property is not the SI one.
For example, when initiliazing `Network` as follows:
```python
# THE PLANT
# let's create a network object with water as the fluid
fluid_list = ['HEOS::Water']
my_plant = Network(fluids=fluid_list)
# set the unit system for temperatures to °C and for pressure to bar
my_plant.set_attr(
    T_unit='C', 
    p_unit='bar', 
    h_unit='kJ / kg',
    m_range=[0.0, 3.0],
    p_range=[1, 16]
)
```
The logs file shows (_pressure unit is not consistent with the values_):
```
...
2023-05-17 11:17:17,040 - DEBUG - network - Setting pressure unit: bar.
...
2023-05-17 11:17:17,040 - DEBUG - network - Setting pressure limits
min: 100000.0 bar
max: 1600000.0 bar
...
````
The log files shows the limit range in SI unit. A quick fix, in lines 340-to-346 in `/network.py` would be either:

1.  Show the range in SI unit regardless of the unit specified by the user (which correspond to the values shown currently):
```python
limits = self.get_attr(prop + '_range_SI')
msg = (
  'Setting ' + fpd[prop]['text'] +
  ' limits\nmin: ' + str(limits[0]) + ' ' +
  fpd[prop]['SI_unit'] + '\n'
  'max: ' + str(limits[1]) + ' ' +
  fpd[prop]['SI_unit'] 
)
logger.debug(msg)
```
2.  Show the range in the unit specified by the user:
```Python
limits = self.get_attr(prop + '_range')
```

**Comment by fwitte** [comment]
Hi @tboussaid,

thanks a lot for letting me know and your suggestions. Are you familar with pull request, and do you want to open one yourself and contribute the changes? If not I can do that no problem. Just let me know.

Thank you for your contribution and have a nice weekend

Francesco

**Comment by tboussaid** [comment]
Hi @fwitte,
Thanks for your reply. Yes I am, I'll do that by the end of the day.
Best,
Taha

**Comment by fwitte** [comment]
Resolved in #420
